Mathlib/GroupTheory/Perm/Basic.lean

```diff
@@ -404,6 +404,16 @@ theorem subtypePerm_ofSubtype (f : Perm (Subtype p)) :
     subtypePerm (ofSubtype f) (mem_iff_ofSubtype_apply_mem f) = f :=
   Equiv.ext fun x => Subtype.coe_injective (ofSubtype_apply_coe f x)
 
+theorem ofSubtype_subtypePerm_of_mem {p : α → Prop} [DecidablePred p]
+    {g : Perm α} {hg : ∀ (x : α), p x ↔ p (g x)}
```

Comment:
Why is `hg` implicit?

Reply:
I can make it explicit, but since it appeared in the goal, I had the idea that it could be inferred.

Reply:
Ah that's possible. Do as you think it is more convenient.